src/sourmash/cli/multigather.py

```diff
@@ -83,6 +83,11 @@ def subparser(subparsers):
     )
     subparser.set_defaults(fail_on_empty_database=True)
 
+    subparser.add_argument(
```

Comment:
This is also appreciated: took me a minute to realize it just spit them out in the $CWD

Reply:
yep 😆

Reply:
(regarding parallelizing - super easy with snakemake, I think, just ask!)